content/applications/websites/forum.rst

```diff
@@ -0,0 +1,227 @@
+   :align: center
+   :alt: create a new forum
+
+Fill in the following information:
+
+- :guilabel:`Forum Name`: add the name of your forum;
+- :guilabel:`Forum Mode`: select :guilabel:`Questions` (only one answer allowed) or
+  :guilabel:`Discussions` (multiple answers allowed);
+- :guilabel:`Privacy`: select :guilabel:`Public` (everyone can access it), :guilabel:`Signed In`
+  (your forum is visible for signed in users only), or :guilabel:`Some users` (your forum and its
+  content is hidden for non-members of selected group).
+
+Click :guilabel:`SAVE`.
+
+.. tip::
+   You can also create a forum from :menuselection:`Website --> Configuration --> Forum: Forums` by
+   clicking the :guilabel:`New` button.
+
+New post creation
+=================
+
+.. note::
+   Only logged-in users can post questions and answer existing ones to avoid one-time participants
+   and spam.
+
+To create a new post, click :guilabel:`New Post` and fill in the following information:
+
+- :guilabel:`Title`: add your question or the topic of your post;
+- :guilabel:`Description`: add a description for your question. Type "/" to use a command and open
+  the :ref:`powerbox <odoo-editor/powerbox>`;
+- :guilabel:`Tags`: add :ref:`tags <forum/tags>` to help filter questions and answers related to the
+  same topic.
+
+Click :guilabel:`Post Your Question`. A new window pops up inviting you to share your question on
+social networks.
+
+Managing posts
+==============
+
+Go to the :guilabel:`Forum` page on your website and select the forum you want to access. From
+there, you can access the following using the forum's navigation bar:
+
+- :guilabel:`Topics`: :guilabel:`All` topics are displayed by default, but you can refine your
+  search by selecting only :guilabel:`Solved`, :guilabel:`Unsolved` or :guilabel:`Unanswered` topics.
+- :guilabel:`People`: display users and their :ref:`Karma gains <forum/karma-gains>` (=XP);
+  :guilabel:`Badges`, and :guilabel:`Certifications`;
+- :guilabel:`Tags`: see the tags used and retrieve questions and discussions based on their tags;
+- :guilabel:`Badges`: in addition to building credibility through questions and answers, active
+  contributors can be rewarded with :ref:`badges <forum/badges>` according to their participation.
+  Badges are visible on both their profile page and their posts;
+- :guilabel:`About`: provide guidelines to answer any questions users might have. By default,
+  guidelines are available and can be edited according to your need by clicking the :guilabel:`Edit`
+  button.
+
+.. _forum/interacting:
+
+Interacting with posts
+======================
+
+From a post, users with enough :ref:`Karma gains <forum/karma-gains>` can :guilabel:`answer`,
+:guilabel:`comment` and :guilabel:`share` a it on social networks.
+
+They can also :guilabel:`Edit`, :guilabel:`Close`, :guilabel:`Delete`, or :guilabel:`Flag` a post by
+clicking ⋮, and :guilabel:`Follow` or :guilabel:`Unfollow` it by clicking the bell.
```

Comment:
I would re-work this section.
Some options are not presented (upvote, downvote, for example)
Some are moderation tools (flag)
Some are not linked to karma points (share).

Presenting karma points and rights way before would make more sense since many functionalities depend on them. For example, the commands ("powerbox") are linked to karma rights.

Reply:
I updated the content as per your comments. We can still discuss it :)